MetaMask/metamask-extension
# Ux Multichain: Hide search input for single account

This PR is to hide the "Search Accounts" input in the Accounts menu if the user has only one account.

## Screenshots/Screencaps

### Before

![Screenshot 2023-04-25 at 7 34 21 PM](https://user-images.githubusercontent.com/39872794/234301830-c4c6fbda-1348-4269-8682-d976efab73bc.jpg)


### After
![Screenshot 2023-04-25 at 7 32 16 PM](https://user-images.githubusercontent.com/39872794/234301252-1db552fb-7b2e-4625-a9a0-c4272984670f.jpg)

## Manual Testing Steps

- Open the account list menu
- Search input field should not be there for single account

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
